Tests fail in TestModelFieldConversion.test_sets_enum_choices with python 2.7 and 3.4

```
$ EPYTHON=python2.7 py.test
============================================================= test session starts ==============================================================
platform linux2 -- Python 2.7.12, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/tiziano/work/gentoo/marshmallow-sqlalchemy, inifile: 
plugins: mock-0.11.0, cram-0.1.1, capturelog-0.7, xdist-1.14, flaky-3.1.0, hypothesis-3.1.0, cov-2.2.1
collected 70 items 

tests/test_marshmallow_sqlalchemy.py ......F...............................................................
===Flaky Test Report===


===End Flaky Test Report===

=================================================================== FAILURES ===================================================================
_______________________________________________ TestModelFieldConversion.test_sets_enum_choices ________________________________________________

self = <tests.test_marshmallow_sqlalchemy.TestModelFieldConversion instance at 0x7fc50d54cc20>
models = <tests.test_marshmallow_sqlalchemy._models object at 0x7fc50d516a10>

    def test_sets_enum_choices(self, models):
        fields_ = fields_for_model(models.Course)
        validator = contains_validator(fields_['level'], validate.OneOf)
        assert validator
>       assert validator.choices == ('Primary', 'Secondary')
E       assert ['Primary', 'Secondary'] == ('Primary', 'Secondary')
E         Use -v to get the full diff

tests/test_marshmallow_sqlalchemy.py:311: AssertionError
============================================ 1 failed, 69 passed, 1 pytest-warnings in 2.70 seconds ============================================
```

respectively

```
$ EPYTHON=python3.4 py.test 
============================================================= test session starts ==============================================================
platform linux -- Python 3.4.5, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/tiziano/work/gentoo/marshmallow-sqlalchemy, inifile: 
plugins: mock-0.11.0, cram-0.1.1, capturelog-0.7, xdist-1.14, flaky-3.1.0, hypothesis-3.1.0, cov-2.2.1
collected 70 items 

tests/test_marshmallow_sqlalchemy.py ......F...............................................................
===Flaky Test Report===


===End Flaky Test Report===

=================================================================== FAILURES ===================================================================
_______________________________________________ TestModelFieldConversion.test_sets_enum_choices ________________________________________________

self = <tests.test_marshmallow_sqlalchemy.TestModelFieldConversion object at 0x7f48b98da438>
models = <tests.test_marshmallow_sqlalchemy.models.<locals>._models object at 0x7f48b99000f0>

    def test_sets_enum_choices(self, models):
        fields_ = fields_for_model(models.Course)
        validator = contains_validator(fields_['level'], validate.OneOf)
        assert validator
>       assert validator.choices == ('Primary', 'Secondary')
E       assert ['Primary', 'Secondary'] == ('Primary', 'Secondary')
E         Use -v to get the full diff

tests/test_marshmallow_sqlalchemy.py:311: AssertionError
============================================ 1 failed, 69 passed, 1 pytest-warnings in 3.13 seconds ============================================
```

... with latest SQLAlchemy-1.1.1 and Marshmallow-2.10.3.

The succeed with Python-3.5

--- a/tests/test_marshmallow_sqlalchemy.py
+++ b/tests/test_marshmallow_sqlalchemy.py
@@ -308,7 +308,7 @@ class TestModelFieldConversion:
         fields_ = fields_for_model(models.Course)
         validator = contains_validator(fields_['level'], validate.OneOf)
         assert validator
-        assert validator.choices == ('Primary', 'Secondary')
+        assert list(validator.choices) == ['Primary', 'Secondary']
 
     def test_many_to_many_relationship(self, models):
         student_fields = fields_for_model(models.Student)

